packages/client/base/src/services/embed/crossmintIFrameService.ts

```diff
@@ -91,3 +93,10 @@ export function crossmintIFrameService(props: CrossmintEmbeddedCheckoutProps) {
         emitInternalEvent,
     };
 }
+
+function coerceCollectionIdToClientId(props: CrossmintEmbeddedCheckoutProps) {
+    if ("collectionId" in props && props.collectionId) {
```

Comment:
Can you explain this check? not sure how the && isnt redundant

Reply:
its mainly a ts thing, but empty string, null would satisfy `"collectionId" in props` so the && check prevents those